code/modules/power/supermatter/supermatter_delamination/delamination_effects.dm

```diff
@@ -82,17 +82,17 @@
 /datum/sm_delam/proc/effect_explosion(obj/machinery/power/supermatter_crystal/sm)
 	explosion(
 		epicenter = get_turf(sm),
-		devastation_range = calculate_explosion(sm) * 0.5, // max 17.5
-		heavy_impact_range = calculate_explosion(sm) + 2, // max 37
-		light_impact_range = calculate_explosion(sm) + 4, // max 39
-		flash_range = calculate_explosion(sm) + 6, //max 41
+		devastation_range = calculate_explosion(sm) * 0.5, // 17.5 at 75 MINUTES
```

Comment:
```suggestion
		devastation_range = calculate_explosion(sm) * 0.5, // 17.5 at 45 MINUTES
```
45 right?

Reply:
I should actually replace that with the define probably.